### PR TITLE
Fixed #24490 -- Set LogEntry.change_message when adding an object.

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

**Comment by collinanderson** [comment]
buildbot, add to whitelist.

File: django/contrib/admin/options.py [hunk 1]
```diff
@@ -708,7 +708,8 @@ def log_addition(self, request, object):
             content_type_id=get_content_type_for_model(object).pk,
             object_id=object.pk,
             object_repr=force_text(object),
-            action_flag=ADDITION
+            action_flag=ADDITION,
+            change_message=message
```
Comment (charettes) [inline django/contrib/admin/options.py]: Add a trailing comma.

Comment (kaedroho) [inline django/contrib/admin/options.py]: That would make this inconsistant with https://github.com/django/django/pull/4335/files#diff-3c42de3e53aba87b32c494f995a728dfR728 and https://github.com/django/django/pull/4335/files#diff-3c42de3e53aba87b32c494f995a728dfR744. Shall I add a trailing comma on these lines as well?

Comment (charettes) [inline django/contrib/admin/options.py]: I think it wouldn't hurt to consistently add it there too.

Comment (kaedroho) [inline django/contrib/admin/options.py]: Done, thanks!

File: tests/admin_views/tests.py [hunk 2]
```diff
@@ -1544,6 +1544,17 @@ def test_add_view(self):
         self.assertEqual(mail.outbox[0].subject, 'Greetings from a created object')
         self.client.get(reverse('admin:logout'))
 
+        # Check that the addition was logged correctly
```
Comment (timgraham) [inline tests/admin_views/tests.py]: It would be nice if the test verified the change message for a form with formsets.

**Comment by timgraham** [comment]
merged in aea02ddfb7c610db9a7cb291b113d6e561d8eba9, thanks!

**Pull request closed without merging**